lambda_function.py

```diff
@@ -9,7 +10,6 @@
 from dateutil.parser import parse
 import uuid
 import os
-from noid.pynoid import mint
```

Comment:
Please remove https://github.com/vt-digital-libraries-platform/S3toDDB/pull/6/files#diff-363d481fe07d1094db2338998d381b71R5-R7 and https://github.com/vt-digital-libraries-platform/S3toDDB/pull/6/files#diff-363d481fe07d1094db2338998d381b71R49-R50, since you don't need Lambda layer anymore

Reply:
Removed LambdaLayer from template.yaml and readme.md. I thought we still need it.